docs/topics/land/land.md

```diff
@@ -0,0 +1,369 @@
+---
+description: Land
+---
+
+# Land
+
+## Introduction
+
+A LAND is a digital piece of real-estate in The Sandbox's metaverse. Game designers use LAND to build digital experiences, such as games or dioramas, and also populate it with ASSETs.
+
+<!-- add links to LAND, ASSET -->
```

Comment:
add links to LAND, ASSET

Reply:
`dioramas`?

Reply:
what link to LAND?

Reply:
sandbox user doc
https://sandboxgame.gitbook.io/the-sandbox/land/what-is-land